pkg/api/vlabs/validate.go

```diff
@@ -1677,9 +1677,6 @@ func (a *Properties) validateContainerRuntime() error {
 
 	// TODO: These validations should be relaxed once ContainerD and CNI plugins are more readily available
 	if containerRuntime == Containerd && a.HasWindows() {
```

Comment:
Should we add any validation around k8s version here?
Will containerd work with versions older than 1.18?

Reply:
I'm not sure. Who could I ask?

Reply:
I think we'll need to try it out 🤣